tools/topology/sof-tgl-nocodec-1ch.m4

```diff
@@ -0,0 +1,186 @@
+#
+# Topology for generic Tigerlake board with no codec and digital mic array.
+#
+# TGL Host GW DMAC support max 6 playback and max 6 capture channels so some
+# pipelines/PCMs/DAIs are commented out to keep within HW bounds. If these
+# are needed then they can be used provided other PCMs/pipelines/SSPs are
+# commented out in their place.
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`ssp.m4')
+include(`pipeline.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Tigerlake DSP configuration
+include(`platform/intel/tgl.m4')
+# include(`platform/intel/dmic.m4')
+
+#
+# Define the pipelines
+#
+# PCM0 <---> volume <----> SSP0
+# PCM1 <---> Volume <----> SSP1
+# PCM2 <---> volume <----> SSP2
+# PCM6 <---- volume <----- DMIC6 (DMIC01)
+#
+
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     period, priority, core,
+dnl     pcm_min_rate, pcm_max_rate, pipeline_rate,
+dnl     time_domain, sched_comp)
+
+# Low Latency playback pipeline 1 on PCM 0 using max 2 channels of s16le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+#PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+#	1, 0, 8, s16le,
+#	1000, 0, 0,
+#	48000, 48000, 48000)
+
+# Low Latency capture pipeline 2 on PCM 0 using max 2 channels of s16le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
```

Comment:
can you fix all comments to describe support for 8 channels? It's all 'max 2 channels of s16le'